### PR TITLE
fix: fix style conflicts

### PR DESCRIPTION
**Pull request by emilyjablonski**
## Issue Overview

This PR addresses #44

## Description

* Renames --bloom variables to --seeds
* Moves the global button overrides to the two individual components that have buttons (am somewhat concerned about all of our CSS resets, considering consumers who might be using this component library with other packages / custom content)
* Updates our root component classes to have a `seeds` prefix ie moving from `.button` --> `.seeds-button`

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have exported any new components
- [x] My commit message(s) and PR title are polished in the conventional commit format, and any breaking changes are indicated in the message and are well-described

### CODE REVIEW

File: src/global/app-css.scss [hunk 1]
```diff
@@ -59,7 +59,7 @@ figure {
   letter-spacing: inherit;
 }
 
-:where(button) {
+:where(button):not(.button) {
```
Comment (emilyjablonski) [inline src/global/app-css.scss:62]: Here is the only other change

File: src/layout/__stories__/Grid.stories.tsx [hunk 2]
```diff
@@ -75,7 +75,11 @@ export const gridExample = () => (
       </Grid.Row>
     </Grid>
 
-    <style>{".grid {  margin-block: var(--bloom-spacer-content); } .grid-row > * { background: #eee; padding: .5rem }"}</style>
+    <style>
+      {
+        ".grid {  margin-block: var(--seeds-spacer-content); } .grid-row > * { background: #eee; padding: .5rem }"
```
Comment (jaredcwhite) [inline src/layout/__stories__/Grid.stories.tsx:80]: Should change to `.seeds-grid` etc. (and other cases below)

**Review by jaredcwhite — APPROVED** [review]
I just saw one missed search-n-replace, otherwise looks good! I think it'll need a refresh to incorporate latest updates on main, and then LGTM

**Review by ludtkemorgan — APPROVED** [review]
🧈

**Pull request merged**